### PR TITLE
Version 0.2

### LINKED ISSUE #1: [feature] Implement simple mode for the generator
Per the discussion in wechat

Key insight: The comments with HTTP(s) links are usually the content that we are interested.

Solution: Add simple mode for generator. We do not keep the structure and just aggregate all contents with links.

/cc @codeworm96 @at15

### PR DESCRIPTION
**Pull request by codeworm96**
Closes #1

### CODE REVIEW

**Comment by at15** [comment]
It's better to have a full example in the help command, but it seems clap does not support example? 

I tried the following command, I think the error is from parameters I passed, though the error message is too brief, it looks like a network error ...

````bash
./target/debug/weekly-gen --output t.txt --issue 73 --repo dyweb/weekly
Error while fetching
`````

Update, my bad, weekly/73 is actually issue 22 .... /w\ but the error message didn't tell me the issue is not found /w\, change it to 22 and it worked

`````bash
./target/debug/weekly-gen --output t.txt --issue 22 --repo dyweb/weekly
`````

And the result is print to both stdout and the file, stdout does not have YAML frontmatter though ...

(根本看不懂 rust 我还是去玩 gal 吧...

**Comment by at15** [comment]
最新一期的效果如下 https://github.com/dyweb/weekly/issues/22

https://github.com/Shopify/ghostferry live mysql migration, and it has tlaplus https://github.com/Shopify/ghostferry/tree/master/tlaplus
https://github.com/dmlc/gluon-nlp #炼丹
http://www.infoq.com/cn/articles/Kubernetes-system-architecture-introduction

所以Kubernetes目前是完全社区化管理吗？好像群里有active contributor? 
https://github.com/tensorlang/tensorlang a differentiable programming language based on TensorFlow

貌似是加了一层  wrapper 的 DSL .... cc @sxjscience 
elastico open sourced x-pack https://github.com/elastic/elasticsearch/tree/master/x-pack including security, alerting, monitoring, graph and machine learning. It was a paid only feature. https://www.elastic.co/products/x-pack/open
https://github.com/japaric/criterion.rs Statistics-driven Microbenchmarking in Rust which is inspired by https://github.com/bos/criterion/
https://github.com/dmlc/gluon-cv #炼丹
http://gluon-cv.mxnet.io/ 
https://github.com/pshihn/rough

Rough.js is a light weight (9kB) graphics library that lets you draw in a sketchy, hand-drawn-like, style.


https://blog.github.com/2018-04-19-introducing-github-learning-lab/

GitHub 出品的，通过 Bot 学习 GitHub 使用的教程
https://github.com/projecteru2/white-paper and https://github.com/laincloud/white-paper container orchestration systems ... cc @gaocegege 听说过没？今天水知乎看到了 ....
https://github.com/jindongwang/transferlearning-tutorial 迁移学习简明手册
https://github.com/tensorflow/swift 这个组合真是奥妙重重

除了 docs 就是图，代码在这里 https://github.com/google/swift/tree/tensorflow
https://www.youtube.com/playlist?list=PLrEMgOSrS_3fghr8ez63xKFNF-BA6Pz13 writing system software by Redis author cc @codeworm96 #就是不炼丹
一些配色网站，在折腾 https://github.com/dyweb/Ayi/pull/79 的时候找了一下 (知乎回答搬运 #pet老师我要做设计

- http://colormind.io/ 可以传图取配色，还有 boostrap 网站的配色生成 ...

>  Colormind is a color scheme generator that uses deep learning. It can learn color styles from photographs, movies, and popular art.

- https://color.adobe.com/create/color-wheel adoubi 
- http://nipponcolors.com/ 日系配色
- http://paletton.com 并不太懂盘子旁边的方块是什么意思
- https://coolors.co 挺好玩，不会用
https://github.com/voxmedia/meme 

Meme is a generator that Vox Media uses to create social sharing images. See working version at http://www.sbnation.com/a/meme.


https://github.com/reproto/reproto a schema manager in Rust cc @codeworm96 作者以前是在 spotify 写 tsdb 的 ... /w 我还给他开过一个策策式的 PR (typo only fix
https://github.com/codeworm96/dy-weekly-generator/pull/3 weekly generator 0.2.0 is ready for reviews!

File: src/bin/weekly-gen.rs [hunk 1]
```diff
@@ -32,7 +32,7 @@ fn main() {
     match work() {
         Err(Error::ConfigErr) => println!("Invalid arguments!"),
         Err(Error::RequestErr(e)) => println!("Error while sending request ({:?})", e),
-        Err(Error::FetchErr) => println!("Error while fetching"),
+        Err(Error::FetchErr(r)) => println!("Error while fetching ({:#?})", r),
```
Comment (at15) [inline src/bin/weekly-gen.rs]: /w\

**Comment by gaocegege** [comment]
可以做一下 section 么，比如一个 comment 算一个 section

**Comment by gaocegege** [comment]
🎉 很硬核，我喜欢=。=

**Comment by codeworm96** [comment]
什么是 section 啊

**Comment by at15** [comment]
@gaocegege 策策请回答喵喵虫的问题

**Comment by gaocegege** [comment]
不好意思，劳动节没有在劳动。就是按照 comment 划分一下？这样看上去就一个 big list 看上去很累 =。= 当然，只是一个小建议

**Comment by at15** [comment]
@codeworm96 其实我觉得最好是生成的 yaml 能多包含一些信息，然后展示和数据分开，浏览的部分可以直接 filter, 比如按语言, comment author, topic etc.  like https://xephonhq.github.io/awesome-time-series-database/?language=Java&backend=All 

当然我要求一向过多，反正写不出来 ... /w\

**Comment by codeworm96** [comment]
@gaocegege 划分举个例子？

**Comment by gaocegege** [comment]
```
// Comment 1的内容
--------------------
// Comment 2 的内容
--------------------
// Comment 3 的内容
--------------------
// Comment 4 的内容
```

**Comment by codeworm96** [comment]
@at15 太难了，需要会自然语言处理的帮我...
(如果大家都按照某种格式写，还是可做的...

**Comment by gaocegege** [comment]
我感觉走着走着又回到了结构化的老路

**Comment by codeworm96** [comment]
这样会被解析成 heading...

**Comment by gaocegege** [comment]
我就举个例子，或者


++++++++++++

**Pull request merged**